contrib/inventory/cloudforms.py

```diff
@@ -22,7 +22,7 @@
 
 from __future__ import print_function
 import argparse
-import ConfigParser
+from six.moves import configparser as ConfigParser
```

Comment:
Shippable is complaining with:
```
00:29 ERROR: contrib/inventory/cloudforms.py:25:1: use `ansible.module_utils.six` instead of `six` (100%)
```

Reply:
hmm...

```
$ python
Python 2.7.11 (default, Oct 17 2016, 14:59:40) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.2 (clang-700.1.81)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from ansible.module_utils.six.moves import ConfigParser
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name ConfigParser
```

That's with Ansible `devel`. Any ideas?

Reply:
oh wait, never mind

```python
from ansible.module_utils.six.moves import configparser as ConfigParser
```

that should work fine.